plugins/ccgx-dmc/fu-ccgx-dmc-devx-device.c

```diff
@@ -216,6 +221,16 @@ fu_ccgx_dmc_devx_device_type_to_name(FuCcgxDmcDevxDeviceType device_type)
 		return "HX3 PD";
 	if (device_type == FU_CCGX_DMC_DEVX_DEVICE_TYPE_DMC_PD)
 		return "DMC PD";
+	if (device_type == FU_CCGX_DMC_DEVX_DEVICE_TYPE_CCG6)
+		return "CCG6";
+	if (device_type == FU_CCGX_DMC_DEVX_DEVICE_TYPE_CCG6SF)
+		return "CCG6SF";
```

Comment:
Given these are shown to the user as the device name, can we expand out what SF stands for?

Reply:
SF means single port and has FET built-in. IC's part number shows SF, so we use SF as default. Maybe keep CCG6SF would be easier for user whom wants to search related data by name.

Reply:
Hmm, so maybe not useful to the end user. Thanks for the reply.